cachi2/interface/cli.py

```diff
@@ -346,6 +355,66 @@ def inject_files(
     )
 
 
+def _prevalidate_sbom_files_args(sbom_files_to_merge: Paths) -> Paths:
+    def enough_files_for_merge(sbom_files_to_merge: Paths) -> Paths:
+        if len(sbom_files_to_merge) < 2:
+            raise typer.BadParameter("Need at least two different SBOM files")
+        return sbom_files_to_merge
+
+    def all_files_are_jsons(sbom_files_to_merge: Paths) -> Paths:
+        for sbom_file in sbom_files_to_merge:
+            try:
+                json.loads(sbom_file.read_text())
+            except ValueError:
+                raise typer.BadParameter(f"{sbom_file} does not look like a SBOM file")
+        return sbom_files_to_merge
+
+    return all_files_are_jsons(enough_files_for_merge(list(set(sbom_files_to_merge))))
+
+
+@app.command()
```

Comment:
I wonder if we want to have this command as `hidden`. It strikes me that it will only serve a specific use case for Konflux, and probably is not something other users would care about. Although there is no hurt in exposing it, but in that case, we probably should have some small section about it in the repository docs.

Reply:
@MartinBasti @eskultety Thoughts?

Reply:
I don't like hidden commands :D

Reply:
I'll add the docs section and will wait for more votes on hidden vs public. I have no strong opinion at the moment, but am leaning towards public command: it looks like it would be less confusing to have it exposed.

Reply:
I don't have strong preferences either, but it seems folks prefer it public. So let's just add a small docs section about it and leave it public.

Reply:
Done.